components/esm/loadcell.hpp

```diff
@@ -78,18 +78,29 @@ struct Cell
 
   struct DATAstruct
   {
-    int mFlags;
-    int mX, mY;
+      int mFlags;
+      int mX, mY;
+
+      DATAstruct() : mFlags(0), mX(0), mY(0)
+      {}
   };
 
   struct AMBIstruct
   {
-    Color mAmbient, mSunlight, mFog;
-    float mFogDensity;
+      Color mAmbient, mSunlight, mFog;
+      float mFogDensity;
+
+      AMBIstruct() : mAmbient(0),
+                     mSunlight(0),
+                     mFog(0),
+                     mFogDensity(0.f)
+      {}
   };
 
   Cell() : mName(""),
            mRegion(""),
+           mData(DATAstruct()),
```

Comment:
These lines looks pointless. A struct will be initialised via the default constructor anyway, even if you don't list it in the initialiser list.

Reply:
I removed this commit.